java-client/src/main/java/org/opensearch/client/opensearch/OpenSearchHttpClient.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.client.opensearch;
+
+import java.io.IOException;
+import java.net.http.HttpResponse;
```

Comment:
We probably need something like `ResponseBase` (similarly to `RequestBase`), definitely we should not bring  HttpClient from standard library here.